powertools-idempotency/src/test/java/software/amazon/lambda/powertools/idempotency/internal/IdempotencyAspectTest.java

```diff
@@ -133,14 +142,43 @@ public void secondCall_inProgress_shouldThrowIdempotencyAlreadyInProgressExcepti
                 DataRecord.Status.INPROGRESS,
                 Instant.now().plus(356, SECONDS).getEpochSecond(),
                 JsonConfig.get().getObjectMapper().writer().writeValueAsString(b),
-                null);
+                null,
+                OptionalLong.of(Instant.now().toEpochMilli() + 1000));
```

Comment:
Can you introduce a variable here too please `Instant.now().toEpochMilli() + 1000`

Reply:
Done